examples/sig-example.cpp

```diff
@@ -252,4 +298,30 @@ int main(void)
      */
     edDsaVerify(edPrivKey, signature, message);
     /*********************************************/
+
+#ifdef DILITHIUM_ENABLED
+    /************** dilithium signature **********/
+    /* Please note that the interface might change slightly once openssl officially supports
+     * dilithium
```

Comment:
Unfortunately I don't know how the openssl interface will look like. So most probably there will be some minor changes.
We try to keep the interface as stable as possible, so that recompilation when "auto" is used wherever feasible should be possible.